roles/as_code/lib/head_of_delivery.rb

```diff
@@ -0,0 +1,80 @@
+
+    Whilst we are an agile software development company and truly believe in the benefits of an agile mindset, we are also aware that not all companies have bought in to the process to the same extent. As a result, you will need to be flexible in adapting delivery style to meet the specific needs of each client and we will want to hear about times that you have applied that flexibility previously, and the benefits that it brought.
+
+    Additionally, you will have a good understanding of the pros and cons of using Scrum and other agile methodologies, particularly when it comes to coaching and empowering delivery teams. You should be able to provide examples of when having a Scrum approach has become problematic for a team and how you helped them to overcome those issues.
+
+    As you will be adopting a servant leader approach to management in this role we will want to hear about how you have worked this way within previous teams, particularly focusing on how you have encouraged servant leadership and why you believe it is a beneficial model for team management.
+
+    You should have experience managing a budget effectively, preferably using a T&M approach. We will want to discuss how you have approached budget management in the past and would love to hear of projects that you budgeted which exceeded the customer's expectations in terms of what they received within their financial constraints.
+
+    ## What you'll get from us
+
+    You will take ownership of the delivery function of a company that is rapidly scaling to meet increased customer demand, giving you exposure to a huge variety of enterprise-level, highly complex programmes with ever evolving challenges, whilst getting to work with a wealth of market-leading companies.
+
+    As a technologically driven company that espouses best practice and aims to become a Thought Leader within the world of software delivery, Made Tech will provide you the opportunity to use the most exciting new technologies and to work in an environment where we are always striving to stay on the cutting-edge of market trends and processes.
+
+    You'll have the chance to develop and adapt our agile processes, coaching both internal and external teams on best practice and designing workshops for building capabilities within customer organisations.
+
+    Finally, as we continue to grow rapidly there are a wealth of directions that a driven individual can progress to within the company, depending on their personal interests. 
+
+    ## What is it like to work at Made Tech?
+
+    We are a technology focussed business that values continuous improvement, experimentation, trusting individuals and teams to do the right thing.
+
+    We are a growing business and now is a good time to join us as there is great opportunity to have a massive impact on the business and grow with us too.
+
+    ## Salary
+
+    This role has a salary of £80-95k depending on experience.
+  DESCRIPTION
+
+  salary 80_000..95_000
```

Comment:
Salary band should be progressive rather than overlap. Either reduce top end of Delivery Manager or increase this bottom end.

Reply:
The delivery manager range is £50 - £85. Would reduce top end to £80k there as its a fairly large range already.

May have to introduce a senior delivery manager level for career progression. Assume there will only be one 'head of delivery' required so there would otherwise be no promotion opportunity.